package/Editor/Settings/PluginSettings.cs

```diff
@@ -164,7 +163,7 @@ private void DrawTileSettings()
         /// <returns></returns>
         private static Preset GetPreset(string _GUID)
         {
-            if (_GUID != null)
+            if (_GUID != null && !string.IsNullOrEmpty(_GUID))
```

Comment:
you do not need the _GUID != null if you have the isNullOrEmpty

Reply:
That's true.
Done !